xarray/tests/test_backends.py

```diff
@@ -1320,6 +1321,18 @@ def roundtrip_append(self, data, save_kwargs={}, open_kwargs={},
                          allow_cleanup_failure=False):
         pytest.skip("zarr backend does not support appending")
 
+    def test_roundtrip_consolidated(self):
+        # my understanding is that pytest.mark.skipif is broken here
+        import zarr
+        if LooseVersion(zarr.__version__) <= '2.2':
+            pytest.skip('zarr version 2.3 or greater needed for consolidated')
```

Comment:
It looks like this way of skipping tests is not working on travis:
https://travis-ci.org/pydata/xarray/jobs/459866378#L7637

Any suggestions for how to do this better?

Reply:
How does `LooseVersion(zarr.__version__) <= LooseVersion('2.2')` work?

You can probably also use:

```python
zarr = pytest.importorskip("zarr", minversion="2.2")
```

Reply:
The problem is that version 2.3 is not released yet. So we need something that skips 2.2 but runs on anything > 2.2. (Current version shows up as 2.2.1.dev2.)

Reply:
I see. 2.3 should be out soon right? I would vote for keeping it simple (because life is short) and skipping everything before 2.3 for these tests.

Reply:
Right now I am doing `pytest.importorskip('zarr', minversion="2.2.1.dev2")`. This is nice because it allows the new tests to be run in the zarr-dev travis environment today (https://travis-ci.org/pydata/xarray/jobs/460869777).